src/OrbitQt/orbithistogram.h

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) 2022 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef ORBIT_HISTOGRAM_H_
+#define ORBIT_HISTOGRAM_H_
+
+#include <qevent.h>
+#include <qpainter.h>
+#include <qwidget.h>
+
+#include <optional>
+#include <string>
+
+#include "Statistics/Histogram.h"
+
+class OrbitHistogram : public QWidget {
```

Comment:
How about calling it HistogramWidget? I know all the existing classes are prefixed with Orbit but that's rather legacy and we try to get away from it.